meteor/lib/collections/TimelineDatastore.ts

```diff
@@ -0,0 +1,16 @@
+import { createMongoCollection } from './lib'
+import { registerIndex } from '../database'
+import { TimelineDatastoreEntryId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+export { TimelineDatastoreEntryId }
```

Comment:
It would be good to avoid re-exporting the Id, there is a task in the backlog to remove a lot of these re-exports

Reply:
Same here, it's consistent with the other files in this subfolder. I can't find the task but if there is a good reference / design to follow I'm happy to do so